### ISSUE TITLE
Remove cloudformation from galaxy

### ISSUE THREAD

**Issue by jbardin** [litl/galaxy#264]
Galaxy (the `galaxy` command) tried to manage the physical pools via aws cloudformation. This mixing of configuration, code, and dependency on AWS specifics has made for a large amount of inflexible code, which is also tied to our own internal configurations. Fixes problems like issue #252 

- Drop the stacks package, and move all cloudformation templating into proper config management
- Remove the `galaxy` command entirely.

**Comment by joeshaw** [comment]
there are a handful of commands in galaxy that aren't in commander.  the one i use most often is `pg:psql`.  Having it autofill `DATABASE_URL` into `psql` is pretty handy, but maybe I can find another way to do that.  `app:backup` and `app:restore` are also galaxy-only currently, but were those deprecated in favor of the specialized dump/restore cools?

**Comment by jbardin** [comment]
I think we can move any required functionality into the single command. I don't think those couple things are a reason to keep `galaxy`.

The dump/restore commands can probably be cleaned up to replace the app:backup/restore. They're very similar, it just has some extra state info for the redis/consul conversion.

**Comment by joeshaw** [comment]
Yeah, didn't mean as a reason to keep around galaxy, just that i'd like to make sure we address those before we kill galaxy.
